### PR TITLE
[READY] add links to vacancies to about pages

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Add some links to our vacancies on the jobs site to the about pages

### CODE REVIEW

File: source/localizable/over-ons/_over-ons.de.md.erb [hunk 1]
```diff
@@ -5,3 +5,12 @@ Defacto ist eine hierarchielose Organisation mit einem hohen Maß an Transparenz
...
 Sind Sie jemand mit einer starken Persönlichkeit und stehen Sie zu Ihrer Meinung? Bei Defacto sind wir immer [auf der Suche nach Menschen](http://jobs.defacto.nl){:target="_blank"} die uns inspirieren.
+
+Wir suchen:
+
+* [Consultants (de)](http://jobs.defacto.nl/2016/02/22/consultant.html){:target="_blank"}
+* [Business Developers (de)](http://jobs.defacto.nl/2016/02/22/business-developer.html){:target="_blank"}
+* [Software Developers (de)](http://jobs.defacto.nl/2016/02/22/software-developer.html){:target="_blank"}
+* [Rails Developers](http://jobs.defacto.nl/2016/03/07/rails-developer.html){:target="_blank"}
+* [Frontend Developers](http://jobs.defacto.nl/2016/02/22/front-end-developer.html){:target="_blank"}
+* [DevOps Engineers](http://jobs.defacto.nl/2016/02/22/front-end-developer.html){:target="_blank"}
```
Comment (sn3p) [inline source/localizable/over-ons/_over-ons.de.md.erb]: **DevOps** links to [Front-end Developer](http://jobs.defacto.nl/2016/02/22/front-end-developer.html).
Besides that most of these vacancies are in Dutch, do we wan't to show them on the German website? If so, lets append `(nl)` or `(Dutch)` to them like you do with the German ones.

File: source/localizable/over-ons/_over-ons.nl.md.erb [hunk 2]
```diff
@@ -4,4 +4,15 @@ Defacto kent een platte organisatie en hoge mate van transparantie. De verantwoo
...
 
+### Werken bij Defacto
+
 Ben jij iemand met een persoonlijkheid en steek je je mening niet onder stoelen en banken? Bij Defacto zijn we altijd [op zoek naar mensen](http://jobs.defacto.nl){:target="_blank"} die ons inspireren.
+
+Momenteel zijn we op zoek naar:
+
+* [Rails Developers](http://jobs.defacto.nl/2016/03/07/rails-developer.html){:target="_blank"}
+* [Frontend Developers](http://jobs.defacto.nl/2016/02/22/front-end-developer.html){:target="_blank"}
+* [DevOps Engineers](http://jobs.defacto.nl/2016/02/22/front-end-developer.html){:target="_blank"}
```
Comment (sn3p) [inline source/localizable/over-ons/_over-ons.nl.md.erb]: Same here: **DevOps** links to [Front-end Developer](http://jobs.defacto.nl/2016/02/22/front-end-developer.html).

**Comment by RonaldDijkstra** [comment]
Thank you @sn3p for review. I removed the dutch vacancies from the German website and added the right URL to the DevOps vacancy.

File: source/localizable/over-ons/_over-ons.de.md.erb [hunk 3]
```diff
@@ -11,6 +13,3 @@ Wir suchen:
 * [Consultants (de)](http://jobs.defacto.nl/2016/02/22/consultant.html){:target="_blank"}
```
Comment (sn3p) [inline source/localizable/over-ons/_over-ons.de.md.erb]: Wil niet pietluttig zijn maar is het nodig om hier nog `(de)` achter te zetten? ;)
Verder prima, mergen maar! :+1:

Comment (RonaldDijkstra) [inline source/localizable/over-ons/_over-ons.de.md.erb]: Pietluttig is good! Thanks

**Pull request merged**